src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.TestPlatform</id>
+    <version>15.0.0</version>
+    <title>Microsoft.TestPlatform</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>The cross platform Microsoft Test Platform.</description>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <!-- Add a third party notice file -->
+    <file src="ThirdPartyNotices.txt" target="" />
+    <file src="Intellitrace\**" exclude="**\*.pdb" target="tools\net451" />
```

Comment:
It seems IntelliTrace dependencies are not being placed in Common7\IDE\.. does CC/Fakes/TI works with this folder structure?

Reply:
It is present in Microsoft.Internal.Intellitrace package.